src/net.c

```diff
@@ -224,6 +225,20 @@ static void ipchange_handler(void *arg)
 }
 
 
+static void set_default_ipv6ll(struct network *net)
+{
+	char ifname[32] = "";
```

Comment:
`ifname[32]` is a bit small, would prefer `ifname[64]`, on windows i see interface names like this:
`{B3B5F4C1-231D-4878-8882-2F81737EEA97}`